floodviz/templates/index.html

```diff
@@ -29,6 +24,9 @@
 
 	<script type="text/javascript" src="{{ url_for('static', filename='bower_components/jquery/dist/jquery.min.js') }}"></script>
 	<script type="text/javascript" src="{{ url_for('static', filename='bower_components/bootstrap/dist/js/bootstrap.js') }}"></script>
+	<script type="text/javascript" src="{{ url_for('static', filename='js/hydrograph.js')}}"></script>
```

Comment:
Sorry, this should go after the d3 and nvd3 because the code depends on it. Generally it's best to maintain that kinds of ordering.